azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -0,0 +1,56 @@
+package azkaban.utils;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by wqs on 17/1/11.
+ */
+public class EmailerTest {
```

Comment:
This test is almost identical to the AbstractMailer test. Can you think of a way to consolidate?